prql-compiler/tests/integration/docker-compose.yml

```diff
@@ -1,3 +1,4 @@
+version: "2.4"
```

Comment:
This line seems added by accident?

Reply:
Nope, it's intentional. 

Docker-compose version 3.x was rejecting the file, saying mssql contains entries that are not allowed.

Reply:
That is weird. Do you know what version of docker compose you are using?

- Currently the latest docker compose is 2.18.1 (See https://github.com/docker/compose)
- docker compose 2.X is based on the compose spec, and `version` is deprecated in the compose spec. https://github.com/compose-spec/compose-spec/blob/master/spec.md#version-top-level-element

Reply:
> $ docker-compose --version
> docker-compose version 1.25.0, build unknown

Oh, your docker-compose version is outdated. compose version 1 is no longer supported, and compose version 2 is widely used these days.
compose v1 is written in Python, and compose v2 is fully rewritten in Go as Docker plugin.
Please uninstall compose v1 and install v2.
https://docs.docker.com/compose/install/